app/templates/babelrc

```diff
@@ -1,5 +1,5 @@
 {
   "presets": [
-    "es2015"
+    "@babel/env"
```

Comment:
Does this shorthand work? I suggest typing `@babel/preset-env` instead either way, because that's how it says in the official documentation. I recall that Babel folks even started discouraging shorthands at some point, but I can't find that info. 🤔

Reply:
Looks like `@babel/preset-env` is the right one

Reply:
Fixed in [427abcedb1a63cb38d7c8bd1b4c25dd0b296bfec](https://github.com/yeoman/generator-webapp/pull/733/commits/427abcedb1a63cb38d7c8bd1b4c25dd0b296bfec)